src/api/Abstraction/SimpleLoopbackAbstraction/Midi2.SimpleLoopbackEndpointManager.cpp

```diff
@@ -155,7 +154,7 @@ CMidi2SimpleLoopbackEndpointManager::CreateParentDevice()
 
 _Use_decl_annotations_
 HRESULT 
-CMidi2SimpleLoopbackEndpointManager::CreateEndpoint(_In_ std::wstring const instanceId, _In_ std::wstring const name, _In_ MidiFlow const flow)
```

Comment:
I assume the SAL annotations aren't required for internal methods? If so, I'll work that change in. Note that the name of this abstraction changed (it's in my branch) so there will be other changes there as well.